custom_components/tuya_local/devices/holman_wx8_irrigation_controller.yaml

```diff
@@ -16,7 +16,7 @@ primary_entity:
       type: integer
     - id: 101
       name: programming
-      type: string
+      type: raw
```

Comment:
raw is not a type, us string as originally

Reply:
Reverted. I was testing. DPS 101 doesn't actualy how up in Tuya_local but it is present in the Tuya Development portal.